apps/api/src/index.ts

```diff
@@ -0,0 +1,65 @@
+import cors from "cors";
+import express, { Express, Request, Response } from "express";
+import { createRedisInstance } from "@pulse/redis";
+const app: Express = express();
+import dotenv from "dotenv";
+import { EventEmitter } from "stream";
+EventEmitter.defaultMaxListeners = 100;
+
+dotenv.config();
+
+const redis = createRedisInstance({
+  port: 42573,
+  host: process.env.REDIS_HOST,
+  password: process.env.REDIS_PASSWORD,
+  tls: {
+    rejectUnauthorized: false,
+  },
+});
+
+app.use(
+  cors({
+    // origin is given a array if we want to have multiple origins later
+    origin: "*",
+    credentials: true,
+  }),
+);
+
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+
+app.use("/hook", async (req: Request, res: Response) => {
+  try {
+    const ixName = req.body[0].meta.logMessages[1].split("Instruction:")[1]; // TODO: can not be determined for all transactions, need a new way to get the name
+    const object = {
```

Comment:
Create a package for types in packages and just add a type so that we can keep it consistent in different place

Reply:
added!